crates/sui-core/src/authority.rs

```diff
@@ -1497,6 +1496,7 @@ impl AuthorityState {
         checkpoint_store: Arc<CheckpointStore>,
         prometheus_registry: &Registry,
         pruning_config: &AuthorityStorePruningConfig,
+        objects: &[Object],
```

Comment:
```suggestion
        genesis_objects: &[Object],
```
nit: more specific naming?

Reply:
good point.